File: Latihan_a/Vga.cpp
#include "Ram.cpp"

class Vga {

    private:
        // atribut
        string brand;
        string type;
        int price;

    public:
        // konstruktor
        Vga(){}

        Vga(string brand, string type, int price)
        {
            this->brand = brand;
            this->type = type;
            this->price = price;
        }

        // setter
        void setBrand(string brand)
        {
            this->brand = brand;
        }
        
        void setType(string type)
        {
            this->type = type;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        // getter
        string getBrand()
        {
            return this->brand;
        }
        
        string getType()
        {
            return this->type;
        }

        int getPrice()
        {
            return this->price;
        }

        // destruktor
        ~Vga(){}
};

File: Latihan_b/Python/Vga.py
class Vga:
    
    # atribut
    __brand = ""
    __type = ""
    __price = 0

    
    # konstruktor
    def __init__(self, brand, type, price):
        self.__brand = brand
        self.__type = type
        self.__price = price

    # setter
    def setBrand(self, brand):
        self.__brand = brand
        
    def setType(self, type):
        self.__type = type

    def setPrice(self, price):
        self.__price = price

    # getter
    def getBrand(self):
        return self.__brand
        
    def getType(self):
        return self.__type

    def getPrice(self):
        return self.__price